test/node-api/test_cannot_run_js/binding.gyp

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
This test can be moved to the `test/js-native-api` folder, as all apis it uses are in the header `js_native_api.h`.